### PR TITLE
modifying documentation so the readthedocs page shows ARTEMIS

### PR DESCRIPTION
**Pull request by RevathiJambunathan**

### CODE REVIEW

File: Docs/source/index.rst [hunk 1]
```diff
@@ -1,33 +1,32 @@
 :orphan:
 
-WarpX
+ARTEMIS
 -----
```
Comment (ax3l) [inline Docs/source/index.rst]: `.rst` is a bit strict on alignments:
```suggestion
ARTEMIS
-------
```

File: Docs/source/index.rst [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
 
-WarpX is an advanced **electromagnetic Particle-In-Cell** code.
+ARTEMIS is an advanced **electromagnetic electrodynamics** code that couples the Maxwell's equations implementation in WarpX with clasical models describing quantum behavior of materials used in microelectronics.
```
Comment (ax3l) [inline Docs/source/index.rst]: "electromagnetic electrodynamics" sounds  a bit doubly to me :)
```suggestion
ARTEMIS is an advanced electrodynamics code based on `WarpX <https://ecp-warpx.github.io>`__.
It couples Maxwell's equations with classical models describing quantum behavior of materials used in microelectronics.
```

File: Docs/source/index.rst [hunk 3]
```diff
@@ -1,33 +1,32 @@
...
 
     - Perfectly-Matched Layers (PML)
-    - Boosted-frame simulations
-    - Mesh refinement
+    - Heterogeneous materials
+    - User-defined excitations
+    - Landau-Lifshitz-Gilbert equations for micromagenetics
 
-For details on the algorithms that WarpX implements, see the :ref:`theory section <theory>`.
+For details on the algorithms that artemis implements, see the :ref:`theory section <theory>`.
```
Comment (ax3l) [inline Docs/source/index.rst]:
```suggestion
For details on the algorithms that ARTEMIS implements, see the :ref:`theory section <theory>`.
```

File: Docs/source/index.rst [hunk 4]
```diff
@@ -1,33 +1,32 @@
...
 
-WarpX is a *highly-parallel and highly-optimized code*, which can run on GPUs and multi-core CPUs, and includes load balancing capabilities.
-In addition, WarpX is also a *multi-platform code* and runs on Linux, macOS and Windows.
+ARTEMIS is a *highly-parallel and highly-optimized code*, which can run on GPUs and multi-core CPUs, and includes load balancing capabilities.
+In addition, ARTEMIS is also a *multi-platform code* and runs on Linux, macOS and Windows. ARTEMIS has leveraged the ECP-code WarpX, and is built on ECP-code AMReX.
```
Comment (ax3l) [inline Docs/source/index.rst]:
```suggestion
In addition, ARTEMIS is also a *multi-platform code* and runs on Linux, macOS and Windows. ARTEMIS has leveraged the ECP code WarpX, and is built on the ECP framework AMReX.
```

File: Docs/source/conf.py [hunk 5]
```diff
@@ -166,8 +166,8 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'WarpX', 'WarpX Documentation',
-     author, 'WarpX', 'WarpX is an advanced electromagnetic Particle-In-Cell code.',
+    (master_doc, 'ARTEMIS', 'ARTEMIS Documentation',
+     author, 'ARTEMIS', 'ARTEMIS is an Adaptive mesh Refinement Time-domain ElectrodynaMIcs Solver for modeling and studying the performance of microelectronics.',
```
Comment (RevathiJambunathan) [inline Docs/source/conf.py]:
```suggestion
     author, 'ARTEMIS', 'ARTEMIS is an Adaptive mesh Refinement Time-domain ElectrodynaMIcs Solver for modeling and studying the performance of microelectronics devices.',
```

**Review by ax3l — APPROVED** [review]

**Pull request merged**